modules/oneVideoBidAdapter.js

```diff
@@ -294,6 +307,7 @@ function getRequestData(bid, consentData, bidRequest) {
     }
   }
   if (bid.params.video.e2etest) {
+    utils.logMessage('+++ oneVideoBidAdapter: E2E test mode enabled. \n The following parameters are being overridden by e2etest mode:\n* bidfloor:null\n* width:300\n* height:250\n* mimes: video/mp4, application/javascript\n* api:2\n* site.page/ref: verizonmedia.com\n* tmax:1000');
```

Comment:
Just out of curious, where are these log available.

Reply:
this goes to console in debug mode

Reply:
It logs when pbjs debug mode is active
Either when the url contains `?pbjs_debug=true`
or
```
pbjs.setConfig({
      debug: true
});
```

Reply:
got it Thank you